### PR TITLE
redo #595 + #607

### PR DESCRIPTION
**Pull request by John-A-Davies**
Signed-off-by: John Davies <daviesja@uk.ibm.com>

### CODE REVIEW

File: scripts/configure/zowe-configure.sh [hunk 1]
```diff
@@ -66,7 +66,7 @@ fi
 zluxserverdirectory='zlux-app-server'
 echo "Preparing folder permission for zLux plugins foder..." >> $LOG_FILE
 chmod -R u+w $ZOWE_ROOT_DIR/$zluxserverdirectory/plugins/
-chmod -R u+w $ZOWE_ROOT_DIR/$zluxserverdirectory/deploy/site
```
Comment (stevenhorsman) [inline scripts/configure/zowe-configure.sh]: Why has this been removed?

File: scripts/configure/zowe-configure.sh [hunk 2]
```diff
@@ -99,25 +117,6 @@ chmod a+x "${ZOWE_ROOT_DIR}/scripts/zowe-support.sh"
 echo "Attempting to setup Zowe Proclib ... "
 . $CONFIG_DIR/zowe-configure-proclib.sh
 
-# TODO LATER - same as the above - zss won't start with those permissions,
-sed -e "s#{{root_dir}}#${ZOWE_ROOT_DIR}#" \
-  -e "s#{{zosmf_admin_group}}#${ZOWE_ZOSMF_ADMIN_GROUP}#" \
```
Comment (stevenhorsman) [inline scripts/configure/zowe-configure.sh]: Why has this code block been moved and bits been deleted?

Comment (John-A-Davies) [reply scripts/configure/zowe-configure.sh]: BAU changes happened in between.  Fixing this now.

File: scripts/configure/zowe-configure.sh (hunk 1 above)
Comment (John-A-Davies) [reply scripts/configure/zowe-configure.sh]: It should be back in now.

File: scripts/zowe-parse-yaml.sh [hunk 3]
```diff
@@ -196,93 +201,98 @@ parseConfiguationFile ./zowe-install.yaml
 if [[ $ZOWE_ROOT_DIR == "" ]] 
 then
     ZOWE_ROOT_DIR="~/zowe/$ZOWE_VERSION"
-    echo "  ZOWE_ROOT_DIR not specified:  Defaulting to ~/zowe/$ZOWE_VERSION"
+    echo "  ZOWE_ROOT_DIR not specified:  Defaulting to ~/zowe/$ZOWE_VERSION" >> $LOG_FILE
```
Comment (stevenhorsman) [inline scripts/zowe-parse-yaml.sh]: These errors should be written to the log, but should also come out on stdout so the user can see that their configuration is incorrect

File: scripts/zowe-support.template.sh [hunk 4]
```diff
@@ -40,7 +40,10 @@ if [[ -z "${ZOWE_PREFIX}" ]];then
     echo "The ZOWE_PREFIX environment variable wasn't properly populated during install. Exiting."
     exit
 fi
-
+if [[ -z "${ZOWE_INSTANCE}" ]];then
```
Comment (stevenhorsman) [inline scripts/zowe-support.template.sh]: ZOWE_INSTANCE doesn't seem to be defined in this file, so this if block will always trigger. I also don't see the need for it

File: scripts/zowe-parse-yaml.sh (hunk 3 above)
Comment (John-A-Davies) [reply scripts/zowe-parse-yaml.sh]: OK, I will do that.

**Comment by stevenhorsman** [comment]
A value of ZOWE_INSTANCE is still missing from the default yaml

**Comment by stevenhorsman** [comment]
I'd like to see PR #611 merged in to clean up the code, but approving on the basis that Joe has assured me the community have approved the new component naming scheme implemented in this change

**Pull request merged**